src/Core/System/SalesChannel/Aggregate/SalesChannelType/SalesChannelTypeEntity.php

```diff
@@ -118,32 +76,58 @@ public function setDescriptionLong(?string $descriptionLong): void
         $this->descriptionLong = $descriptionLong;
     }
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:return-type-change - return type will be nullable
+     */
     public function getCoverUrl(): string
     {
+        if ($this->coverUrl === null) {
```

Comment:
maybe it would be good, to add a hint, that this ifs can be removed with 6.8. I am not sure, if we will remember in a year, what to do here :sweat_smile:

Reply:
good point, i will update the deprecations 👍